nav2_mppi_controller/src/critics/cost_critic.cpp

```diff
@@ -26,6 +26,7 @@ void CostCritic::initialize()
   getParam(power_, "cost_power", 1);
   getParam(weight_, "cost_weight", 3.81f);
   getParam(critical_cost_, "critical_cost", 300.0f);
+  getParam(near_collision_cost_, "near_collision_cost", 253.0f);
```

Comment:
Docs update with this would be good (+ migration guide for the new parameter entry)

Reply:
Shouldn't this be an `int` though, not a double (as well in the param files)? This is a costmap cost